cloud-aws-common/build.gradle

```diff
@@ -21,6 +21,9 @@ dependencies {
   implementation project(':common-handlebar')
   implementation project(':template-manager-tag')
   implementation project(':auth-connector')
+  implementation project(':usage-collection')
```

Comment:
As I told before when I delete it the compiler doesn't recognize MeteringEventsProto

Reply:
Pretty weird. Let us leave it then.

Reply:
you need to add to the cloud-api as `api project(':usage-collection')` instead of `implementation project(':usage-collection')` then it win't be needed here ;)

Reply:
Looks like the `api` dependency configuration would need the `java-library` [plugin](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation), so it is likely not worth the effort adding for sake of such transitive project dependencies.